src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -154,7 +154,6 @@ public void AddProject(AbstractProject project)
                 {
                     AssemblyName = project.AssemblyName,
                     FilePath = project.ProjectFilePath,
-                    Hierarchy = project.Hierarchy,
```

Comment:
@tmat @jasonmalinowski What's the new way for TS to thread this through?  Some of our refactorings are broken since the IVsHierarchy isn't being returned by IVsSolution.

As an aside, can you please ping me/us when changing a TS shim?

Reply:
Are you currently calling `VisualStudioProject.GetHierarchy` to get such IVsHierarchy objects?

Reply:
Which `VisualStudioProject`?  I'm assuming you don't mean the one in `Microsoft.VisualStudio.LanguageServices.TypeScript.ScriptContexts`.  Either way, we're trying to specify hierarchies - not retrieve them.